style.css

```diff
@@ -509,3 +522,304 @@ textarea::placeholder {
+    }
+  }
+
+  /* /* CARD1 GLOBAL SETTINGS */
+  .card1 {
+    display: grid;
+    grid-template-columns: auto auto;
+    justify-content: space-evenly;
+    align-content: center;
+    gap: 3%;
+    background-color: white;
+    border: 2px solid #dfe1e6;
+    border-radius: 1em;
+
+    /* border: 3px solid green; */
+    width: 70rem;
+    height: 25rem;
+  }
+
+  /* _______________CARD 1___________ */
+
+  /* CARD1 ALL TEXT SETTING */
+
+  .card .cardWorks {
+    /* border: 1px solid red; */
+    width: 37rem;
+    height: 30rem;
+  }
+
+  /* Img 1 SETTINGS */
+  .cardImg {
+    /* border: 1px solid blue; */
+    width: 37rem;
+    height: 30rem;
+  }
+
+  /* card1 img */
+  .cardImg1 {
+    content: url(assets/bigSizePicCard1.svg);
+  }
+
+  /* card2 img */
+
+  .cardImg2 {
+    content: url(assets/bigSizePicCard2.svg);
+  }
+
+  /* card3 img */
+
+  .cardImg3 {
+    content: url(assets/bigSizePicCard3.svg);
+  }
+
+  /* card4 img */
+  .cardImg4 {
+    content: url(assets/bigSizePicCard4.svg);
+  }
+
+  /* Invert img */
+  .card-container:nth-child(4) .cardImg,
+  .card-container:nth-child(6) .cardImg {
+    order: 1;
+  }
+
+  /* Word TONIC */
+  .cardWorks h4 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-left: 2.2rem;
+  }
+
+  /* Class card work skills setting */
+  .cardWorks-skills {
+    margin-top: 1rem;
+  }
+
+  /* Words "canopy" "back end dev" and "2015" settings */
+  .word1 {
+    width: 43px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .word2 {
+    width: 120px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    padding: 8px;
+    color: #6b778c;
+  }
+
+  .word3 {
+    padding: 8px;
+    width: 38px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  /* settings for pharagarph */
+  .cardWorks .pCards {
+    margin-top: 1rem;
+    width: 31rem;
+    height: 50px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  /* settings for words: html css and javascript */
+  .workContainer {
+    margin-top: 1rem;
+    border-radius: 8px;
+    margin-left: 2.2rem;
+  }
+
+  /* settings for words: html css and javascript */
+  .cardButton {
+    width: 140px;
+    margin-left: 2.2rem;
+    margin-top: 3rem;
+  }
+
+  /* About myself */
+  .aboutMyself {
+    display: flex;
+    flex-direction: row;
+    background-color: white;
+    border-top-right-radius: 6rem;
+    gap: 3rem;
+  }
+
+  .aboutMyself-container {
+    width: 40%;
+    margin-left: 3rem;
+  }
+
+  .aboutMyself-container h3 {
+    margin-top: 2rem;
+  }
+
+  .desktopVersion {
+    width: 45%;
+  }
+
+  .aboutMyself-container p {
+    padding: 0;
+    margin: 0;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 30px;
+  }
```

Comment:
kindly align the technologies listed in the `about me ` section to the left as shown the Figma design given to have a better design

Figma design:
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/20537712/e7a9692e-6fc4-4bed-85a6-33c91bb1536c)

Your design :
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/20537712/b7e3c25b-2895-4285-b064-bb3466e27a2b)